### PR TITLE
[alerts] update permission control

### PR DESCRIPTION
**Pull request by waiterZen**

### CODE REVIEW

File: api/utils/rights.js [hunk 1]
```diff
@@ -956,6 +956,23 @@ exports.getUserApps = function(member) {
...
+        for (var appId in permissionList) {
+            const targetPermissionForApp = permissionList[appId];
+            if (targetPermissionForApp.all === true || targetPermissionForApp.allowed[feature] === true) {
+                userApps.push(appId);
+            }
+        }
+    }
+    return userApps;
+};
+
```
[the remark this answers is not in the document]
Comment (frknbasaran) [reply api/utils/rights.js:975]: Is there any uncovered case that causes we need for this? :) 

even if there is, this method returns an empty array for global admin. it looks wrong to me but maybe there is a reason that I can't see right now. :)

[the remark this answers is not in the document]
Comment (waiterZen) [reply api/utils/rights.js:975]: We need a function like https://github.com/Countly/countly-server/blob/feature/new-ui/api/utils/rights.js#L944
Fetching app list for more specific permission type (C/R/U/D)  for target plugin to assemble some query.

[the remark this answers is not in the document]
Comment (frknbasaran) [reply api/utils/rights.js:975]: hi @waiterZen, thanks for explanation. :) but still, it should not return an empty array when the member is a global admin. it should return basically all apps when the member is a global admin.

[the remark this answers is not in the document]
Comment (waiterZen) [reply api/utils/rights.js:975]: If it is a global admin, I think it is no need to check app user permission?
Similar to https://github.com/Countly/countly-server/blob/feature/new-ui/api/utils/rights.js#L944
and https://github.com/Countly/countly-server/blob/feature/new-ui/api/utils/rights.js#L962

[the remark this answers is not in the document]
Comment (frknbasaran) [reply api/utils/rights.js:975]: oh good point. yes, we don't need to check app permission for global admins. 
just confused when I see that empty array.

**Review by frknbasaran — APPROVED** [review]

**Pull request merged**